http/headers/Cookie.json

```diff
@@ -14,7 +14,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "≤50"
```

Comment:
I think we can confidently say this was supported in Firefox 1 and Safari 1. Cookies were supported in Netscape.

Reply:
I did some more research and determined a few headers that were supported from the beginning.  I opened https://github.com/mdn/browser-compat-data/pull/23619 to handle those!